### ISSUE TITLE
django-fobi 0.10.3 downgrades django-autoslug from 1.9.3 to 1.7.1

### ISSUE THREAD

**Issue by XaviP** [barseghyanartur/django-fobi#60]
Hi @barseghyanartur , thanks for your work in this package.

We've made and upgrade to django-fobi 0.10.3 and we've found that when running:
```
pip install -r requirements.txt --upgrade
```
the django-autoslug is downgraded from 1.9.3 to 1.7.1, which gives an error (not in fobi):
```
AttributeError: 'AutoSlugField' object has no attribute 'allow_unicode'
```
Everything is solved upgrading again django-autoslug to 1.9.3, but any time we try to upgrade pip packages, it happens again.

We don't know exactly if it's because of django-fobi upgrade, there's other packages that has been upgraded too, but at first sight it seems so. Sorry if it's not the case, and feel free to close this issue if so.

**Comment by XaviP** [comment]
btw, we are running django 1.10

**Comment by barseghyanartur** [comment]
@XaviP:

Thanks for reporting this. I'll check it soon.

**Comment by barseghyanartur** [comment]
@XaviP:

It seems to me that hard-coded requirement of ``django-autoslug==1.7.1`` resides is in your own ``requirements.txt``.

Both current branch and 0.10.3 release do install ``django-autoslug`` 1.9.3 when no ``Django`` version is detected or ``Django`` version is greater or equal to 1.9.

**Comment by XaviP** [comment]
I've installed pipdeptree (pip install pipdeptree)
http://stackoverflow.com/a/30450999/3336408
When I run it:
```
(env)$ pipdeptree
[...]
django-fobi==0.11.3
  - django-autoslug [required: ==1.7.1, installed: 1.9.3]
  - django-formtools [required: >=1.0, installed: 2.0]
    - Django [required: >=1.8, installed: 1.10.7]
  - django-nine [required: >=0.1.10, installed: 0.1.12]
  - django-nonefield [required: >=0.1, installed: 0.1]
  - easy-thumbnails [required: >=2.3, installed: 2.4.1]
    - django [required: >=1.4.2, installed: 1.10.7]
    - pillow [required: Any, installed: 4.1.1]
      - olefile [required: Any, installed: 0.44]
  - Pillow [required: >=2.0.0, installed: 4.1.1]
    - olefile [required: Any, installed: 0.44]
  - requests [required: >=1.0.0, installed: 2.14.2]
  - simplejson [required: >=3.0.0, installed: 3.10.0]
  - six [required: >=1.9, installed: 1.10.0]
  - Unidecode [required: >=0.04.1, installed: 0.4.20]
  - vishap [required: >=0.1.5,<2.0, installed: 0.1.5]
    - six [required: >=1.4.1, installed: 1.10.0]
[...]
```
We don't have django-autoslug in our requirements.txt:
https://github.com/FreedomCoop/valuenetwork/blob/master/requirements.txt

**Comment by barseghyanartur** [comment]
@XaviP:

Confirmed. Somehow it happens after 0.10.6. Can't figure out yet what causes it. It will be fixed in the next version.

**Comment by barseghyanartur** [comment]
@XaviP:

I have researched a little bit more. It seems to be a problem of wheel package (or me using it incorrectly).

If you install any version of django-fobi with pip using tar.gz download link (available on PyPI), it goes fine. The setup.py script detects your package versions and uses requirements that are valid for your setup.

Just tried it with files on [PyPI](https://pypi.python.org/pypi/django-fobi):

**django-fobi-0.11.4.tar.gz**

```sh
pip install https://pypi.python.org/packages/a5/27/1784a45a8f3fd6b1f1047a822f761219b4242dbf08c2daa3b815de6632f7/django-fobi-0.11.4.tar.gz
```

**django_fobi-0.11.4-py2.py3-none-any.whl**

```sh
pip install https://pypi.python.org/packages/25/ee/a4a10899c7a221a104c01e8428699d1810da8d973bf6bb7243fbfd135de8/django_fobi-0.11.4-py2.py3-none-any.whl
```

However, in case of wheels it does seem to only use the requirements that we valid on the moment I have been submitting the package to PyPI. Thus, if I have been using the Python3 and Django 1.8, the hard-coded requirement of the whee distribution for django-autoslug goes all the way down to 1.7.1, because I have specified so it in the setup.py script.

I'll have to dive into the [wheels docs](http://wheel.readthedocs.io/en/latest/#defining-conditional-dependencies) to solve this problem properly. For now, I'll make a release using latest django (1.10). Hopefully, everybody else would be happy about it.

I'm releasing an updated version (0.11.5) shortly.
